.github/workflows/pr.yml

```diff
@@ -0,0 +1,47 @@
+---
+name: Validate pull requests
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  schema:
+    name: Validate JSON Schema
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          path: oscal-demo-content
+      - name: Download schemas
```

Comment:
Do we want to pin this to a tag like `1.0.1`?

Reply:
Honestly... even better would be to validate against the `oscal-version` within each document's metadata rather than applying an arbitrary tag to all of them. But that workflow starts to feel rather complex. I also think that all our documents currently specify `1.0.0`.